lib/install/management/appliance.go

```diff
@@ -1076,6 +1076,20 @@ func (d *Dispatcher) ensureApplianceInitializes(conf *config.VirtualContainerHos
 	// as we're only using it for IP in the success case
 	updateErr := d.applianceConfiguration(conf)
 
+	// confirm components launched correctly
```

Comment:
This was added because I managed to break process launch which prevented vic-init from starting endpoint components, but that went completely unreported by vic-machine